lib/cli/package-lock.json

```diff
@@ -1,8 +1,2255 @@
 {
   "name": "@alfresco/adf-cli",
   "version": "5.1.0",
-  "lockfileVersion": 1,
+  "lockfileVersion": 2,
```

Comment:
this file should not be committed I think

Reply:
yep , my mistake. i've done some devel on adf-cli and did not see package-lock was updated